### PR TITLE
Added information about MediaCaptureInitializationSettings properties

### PR DESCRIPTION
**Pull request by strongstrongyiwen**
Added information about what MediaCaptureInitializationSettings properties can be or cannot be configured when ShareMode is set. @kasumman

### CODE REVIEW

**Comment by PRMerger16** [comment]
@strongstrongyiwen : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: windows.media.capture/mediacaptureinitializationsettings_sharingmode.md [hunk 1]
```diff
@@ -18,6 +18,28 @@ An object that specifies the sharing mode for the [MediaCapture](mediacapture.md
 ## -remarks
 Multiple apps can simultaneously acquire frames from the same [MediaFrameSource](../windows.media.capture.frames/mediaframesource.md), but only a single app can acquire exclusive control of and modify the settings for the frame source. Set **SharingMode** to [ExclusiveControl](mediacapturesharingmode.md) if you need to adjust the configuration of any of the frame sources included the requested [SourceGroup](mediacaptureinitializationsettings_sourcegroup.md). However, this means that the call to [InitializeAsync](mediacapture_initializeasync_315323248.md) will fail if another app already has exclusive control of one of the frame sources in the group. If you only need to acquire frames without modifying the configuration, set **SharingMode** to [SharedReadOnly](mediacapturesharingmode.md).
 
+When the **SharingMode** is set to [SharedReadOnly](mediacapturesharingmode.md), some of the MediaCaptureInitializationSettings properties may not be abled to be configured. See below table for details.
```
Comment (kasumman) [inline windows.media.capture/mediacaptureinitializationsettings_sharingmode.md]: "may not be abled to be configured" not sure is this grammatically correct, my ear says no, but I'm not the correct person to do grammar checks :)

**Review by kasumman — COMMENTED** [review]
Add something MediaCaptureInitializationSettings remarks as well

**Comment by PRMerger7** [comment]
@strongstrongyiwen : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger10** [comment]
@strongstrongyiwen : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: windows.media.capture/mediacaptureinitializationsettings_sharingmode.md (hunk 1 above)
Comment (strongstrongyiwen) [reply windows.media.capture/mediacaptureinitializationsettings_sharingmode.md]: changed to "could not be configured", sounds better?

**Comment by Jak-MS** [comment]
@drewbatgit - if you approve this PR, it'll need to be moved out of draft stage and signed off. thanks!

File: windows.media.capture/mediacaptureinitializationsettings.md [hunk 2]
```diff
@@ -26,6 +26,9 @@ For how-to guidance on initializing and shutting down the **MediaCapture** objec
 | 1607 | 14393 | SourceGroup |
 | 1703 | 15063 | AlwaysPlaySystemShutterSound |
 
+> [!NOTE]
```
Comment (kasumman) [inline windows.media.capture/mediacaptureinitializationsettings.md:29]: Is this tag working correctly ? According to this it should https://docs.microsoft.com/en-us/contribute/markdown-reference, is it just github viewer that does not show it properly.

Comment (strongstrongyiwen) [reply windows.media.capture/mediacaptureinitializationsettings.md:29]: I was trying to have some thing like this:
image
And I referenced other .md and it looked just like this.
image

Comment (kasumman) [reply windows.media.capture/mediacaptureinitializationsettings.md:29]: Yeah I think it is correct syntax, github rich preview just does not show it correctly. Keep it!

**Comment by PRMerger7** [comment]
@drewbatgit : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by drewbatgit — APPROVED** [review]

**Pull request merged**